pkg/sanity/sanity.go

```diff
@@ -163,18 +165,30 @@ func newContext(reqConfig *Config) *SanityContext {
 // Test will test the CSI driver at the specified address by
 // setting up a Ginkgo suite and running it.
 func Test(t *testing.T, reqConfig *Config) {
-	path := reqConfig.TestVolumeParametersFile
-	if len(path) != 0 {
-		yamlFile, err := ioutil.ReadFile(path)
+	volParamPath := reqConfig.TestVolumeParametersFile
+	if len(volParamPath) != 0 {
+		yamlFile, err := ioutil.ReadFile(volParamPath)
 		if err != nil {
-			panic(fmt.Sprintf("failed to read file %q: %v", path, err))
+			panic(fmt.Sprintf("failed to read file %q: %v", volParamPath, err))
 		}
 		err = yaml.Unmarshal(yamlFile, &reqConfig.TestVolumeParameters)
 		if err != nil {
 			panic(fmt.Sprintf("error unmarshaling yaml: %v", err))
 		}
 	}
 
+	snapParamPath := reqConfig.TestSnapshotParametersFile
+	if len(snapParamPath) != 0 {
```

Comment:
Instead copying the unmarshaling code from the `reqConfig.TestVolumeParametersFile` handling above, can you move it into a helper function?

Reply:
Thanks for your review. I will amend my commit.